### ISSUE TITLE
Medical icons not included?

### ISSUE THREAD

**Issue by Anastasiia59** [BenjjinF/django-fontawesome-5#18]
Hi! Thank you for this nice package first of all. However, it seems to me that it can't render the icons from 'medical' package - the other simple icons are rendering perfectly fine. Can I do something with it?

**Comment by mcrot** [comment]
I also stumbled on this. I think only the free icons are included. Many more icons are included in the "pro" edition. 
You can see it [here](https://fontawesome.com/icons) when using the filters in the sidebar on the left.

**Comment by Anastasiia59** [comment]
> I also stumbled on this. I think only the free icons are included. Many more icons are included in the "pro" edition.
> You can see it [here](https://fontawesome.com/icons) when using the filters in the sidebar on the left.
> Maybe you want to use a non-free icon?

I don't think so. I tried using 'virus' 'fas' for example, which is apparently a free icon, but it just rendered nothing on the website.

**Comment by mcrot** [comment]
Oh yes, know I see. You're right, `virus` is free and it does not appear.

**Comment by zypro** [comment]
This is caused by an old version of `https://github.com/BenjjinF/django-fontawesome-5/blob/master/fontawesome_5/static/fontawesome_5/css/all.min.css`. 

A fast workaround is to use the current version by setting: `FONTAWESOME_5_CSS = 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css'`.
